implementation/libs/jbb-lib-cache/src/main/java/org/jbb/lib/cache/JbbCacheManager.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2017 the original author or authors.
+ *
+ * This file is part of jBB Application Project.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.jbb.lib.cache;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JbbCacheManager {
+    private final ProxyJCacheManager proxyJCacheManager;
+    private final ProxySpringCacheManager proxySpringCacheManager;
+    private final SpringCacheManagerFactory springCacheManagerFactory;
+
+    @Autowired
+    public JbbCacheManager(ProxyJCacheManager proxyJCacheManager,
+                           ProxySpringCacheManager proxySpringCacheManager,
+                           SpringCacheManagerFactory springCacheManagerFactory) {
+        this.proxyJCacheManager = proxyJCacheManager;
+        this.proxySpringCacheManager = proxySpringCacheManager;
+        this.springCacheManagerFactory = springCacheManagerFactory;
+    }
+
+    public void refresh() {
+        if (!proxyJCacheManager.isClosed()) {
+            proxyJCacheManager.close();
+        }
+
+        proxySpringCacheManager.setCacheManagerBeingProxied(springCacheManagerFactory.build());
```

Comment:
Yep. Here we're setting target object to proxy class. What is your suggestion for naming of this class?